code/main/edge/locationHintManager.brs

```diff
@@ -64,24 +60,40 @@ function _adb_LocationHintManager() as object
                     continue for
                 end if
 
-                if not _adb_isEmptyOrInvalidString(payload.scope) and LCase(payload.scope) = m._EDGE_NETWORK_SCOPE
+                if _adb_stringEqualsIgnoreCase(payload.scope, m._EDGE_NETWORK_SCOPE)
                     m.setLocationHint(payload.hint, payload.ttlSeconds)
                 end if
 
             end for
         end function,
 
+        _setExpiryTime: function(ttlSeconds as dynamic) as void
+            if _adb_isInvalidInt(ttlSeconds)
+                _adb_logDebug("_adb_LocationHintManager::_setExpiryTime() - ttlSeconds is not found, using default ttl (" + FormatJson(m._DEFAULT_LOCATION_HINT_TTL_SEC) + ") seconds.")
```

Comment:
Can we print the invalid `ttlSeconds`? It should be helpful when debugging the issue.